IFC4x3/PropertyEnumerations/t/PEnum_TrackFasteningElasticityType.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_TrackFasteningElasticityType" Name="PEnum_TrackFasteningElasticityType" UniqueId="656edf58-e34e-497b-8de7-90e68ded1ba5">
```

Comment:
Does it require documentation?

Reply:
Done. Commit : de4bcf9de7f5e2e1d9e24e66e3b338da3b5506eb